spinnaker-monitoring-third-party/third_party/prometheus/prometheus.conf

```diff
@@ -1,6 +1,6 @@
 start on filesystem or runlevel [2345]
 
-exec /opt/PROMETHEUS_VERSION/prometheus \
-  -config.file /opt/PROMETHEUS_VERSION/spinnaker-prometheus.yml \
-  -storage.local.path /opt/PROMETHEUS_VERSION/data \
+exec /opt/prometheus/prometheus \
```

Comment:
What's the update story now for migrating old config? Is it assumed users shouldn't touch these config?